zoo/policies/competition_agent.py

```diff
@@ -0,0 +1,40 @@
+import subprocess
+import sys
+import os
+import logging
+import importlib.util
+import shutil
+
+from pathlib import Path, PurePath
+from smarts.core.agent import Agent
+
+
+class CompetitionAgent(Agent):
+    def __init__(self, policy_path, policy):
+        env_name = Path(policy_path).name  # name of the submission file
+        root_path = Path(__file__).parents[2]  # Smarts main path
+
+        self._policy_dir = policy_path
+        self._comp_env_path = str(os.path.join(root_path, "competition_env"))
+        self._sub_env_path = str(os.path.join(self._comp_env_path, env_name))
+
+        self._policy = policy
+
+    def act(self, obs):
+        return self._policy.act(obs)
+
+    def close(self, remove_all_env=False):
```

Comment:
I do not see where this is called.

Reply:
It should be called when all works in done inside ` test_agent.py`. I will modify `test_agent.py` to include this.